### ISSUE TITLE
account button disappears if user's name is too long

### ISSUE THREAD

**Issue by dwrensha** [sandstorm-io/sandstorm#2553]
If I have an identity with a very long name:
<img width="1430" alt="long name" src="https://cloud.githubusercontent.com/assets/495768/18530025/8eeb47d4-7a9d-11e6-8bb0-0dd0e75378d4.png">

then when I resize my window small enough that the name can't fit, the account button totally disappears:

<img width="1285" alt="long name hidden" src="https://cloud.githubusercontent.com/assets/495768/18530026/942fc58a-7a9d-11e6-8771-038ad7bae68a.png">

This seems like a use case for `overflow: ellipsis`.

**Comment by zarvox** [comment]
Yeah, we should probably give a `max-width` to that field (perhaps a percentage of screen width?), and then `@extend %overflow-ellipsis;`
